### DISCUSSION TITLE
Summarize your question into a single sentence here

### DISCUSSION THREAD

**Discussion by Emper-oreDiamond** [GriefPrevention/GriefPrevention#892] [Q&A] [answered]
Hi, thinking about using GriefProtection for my in the works sever, many bases on the server currently contain hostile mobs which contribute to the bases looks, will the GriefProtection plugin do anything to hostile mobs? Will they despawn and will they be protected?

**Comment by Jikoo** [comment, marked as answer]
GP does not remove existing hostiles nor prevent spawning. GP does not protect hostile mobs either - there's no easy way to tell the difference between a caged mob and one that can actually get at people. Most people just fully enclose hostiles they want to keep on display so no one can attack them.
